moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -125,7 +125,9 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
       );
     } else {
       return (
-        <ProjectSummaryMap projectExtentGeoJSON={projectFeatureCollection} />
+        <ProjectSummaryMap
+          projectFeatureCollection={projectFeatureCollection}
+        />
```

Comment:
adapting to the new db schema and mirroring the way we construct `projectFeatureCollection` in `ProjectComponents` and